pyproject.toml

```diff
@@ -802,6 +802,130 @@ testing = ["dev", "providers.tests", "tests_common", "tests", "system", "unit",
+"airflow-core/src/airflow/example_dags/example_assets.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_branch_labels.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_branch_python_dop_operator_3.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_complex.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_custom_weight.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_dag_decorator.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_display_name.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_dynamic_task_mapping.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_dynamic_task_mapping_with_no_taskflow_operators.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_inlet_event_extra.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_kubernetes_executor.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_latest_only_with_trigger.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_local_kubernetes_executor.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_nested_branch_dag.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_outlet_event_extra.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_params_trigger_ui.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_params_ui_tutorial.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_passing_params_via_test_command.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_setup_teardown.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_setup_teardown_taskflow.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_simplest_dag.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_skip_dag.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_task_group.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_task_group_decorator.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_time_delta_sensor_async.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_trigger_target_dag.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_workday_timetable.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_xcom.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/example_xcomargs.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/plugins/event_listener.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/tutorial.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/tutorial_dag.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/tutorial_objectstorage.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/tutorial_taskflow_api.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/tutorial_taskflow_api_virtualenv.py" = ["TID251"]
+"airflow-core/src/airflow/example_dags/tutorial_taskflow_templates.py" = ["TID251"]
```

Comment:
We should just ignore the entre `example_dags` folder

Reply:
Handled in [excluding example dags](https://github.com/apache/airflow/pull/53627/commits/0b5b397532b5974ba7e7c4c8b8d23c880533b80c)